http_parser.c

```diff
@@ -965,9 +965,7 @@ size_t http_parser_execute (http_parser *parser,
           case 'M': parser->method = HTTP_MKCOL; /* or MOVE, MKACTIVITY, MERGE, M-SEARCH, MKCALENDAR */ break;
           case 'N': parser->method = HTTP_NOTIFY; break;
           case 'O': parser->method = HTTP_OPTIONS; break;
-          case 'P': parser->method = HTTP_POST;
-            /* or PROPFIND|PROPPATCH|PUT|PATCH|PURGE */
-            break;
+          case 'P': parser->method = HTTP_POST; /* or PROPFIND, PROPPATCH, PUT, PATCH, PURGE */ break;
```

Comment:
Please apply 80 column limit. I know it is different in other places of the file, but I'd like to enforce this style on all future changes.